bundles/org.openhab.binding.kodi/README.md

```diff
@@ -1,191 +1,198 @@
-| mediatype             | String               | The media type of the current file. Valid return values are: `unknown`, `channel`, `episode`, `movie`, `musicvideo`, `picture`, `radio`, `song`, `video`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
```

Comment:
Github shows the whole file as diff? I don't know why. Maybe line endings are mixed up? Can you check.

Reply:
Thanks for your line endings tip, I commit a new  readme.md